<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DisplaySettingsWidget</class>
 <widget class="QWidget" name="DisplaySettingsWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>485</width>
    <height>525</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QGroupBox" name="basicGroupBox">
     <property name="title">
      <string>Basic</string>
     </property>
     <layout class="QFormLayout" name="formLayout_2">
      <item row="0" column="0">
       <widget class="QLabel" name="label">
        <property name="text">
         <string>Renderer:</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QComboBox" name="renderer"/>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label_5">
        <property name="text">
         <string>Adapter:</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QComboBox" name="adapter"/>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>Fullscreen Mode:</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QComboBox" name="fullscreenMode"/>
      </item>
      <item row="3" column="0" colspan="2">
       <layout class="QGridLayout" name="basicCheckboxGridLayout">
        <item row="0" column="0">
         <widget class="QCheckBox" name="gpuThread">
          <property name="text">
           <string>Threaded Rendering</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QCheckBox" name="threadedPresentation">
          <property name="text">
           <string>Threaded Presentation</string>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QCheckBox" name="vsync">
          <property name="text">
           <string>VSync</string>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QCheckBox" name="syncToHostRefreshRate">
          <property name="text">
           <string>Sync To Host Refresh Rate</string>
          </property>
         </widget>
        </item>
        <item row="2" column="0">
         <widget class="QCheckBox" name="displayAllFrames">
          <property name="text">
           <string>Optimal Frame Pacing</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox_3">
     <property name="title">
      <string>Screen Display</string>
     </property>
     <layout class="QFormLayout" name="formLayout">
      <item row="0" column="0">
       <widget class="QLabel" name="label_4">
        <property name="text">
         <string>Aspect Ratio:</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <layout class="QHBoxLayout" name="horizontalLayout" stretch="1,0,0,0">
        <item>
         <widget class="QComboBox" name="displayAspectRatio"/>
        </item>
        <item>
         <widget class="QSpinBox" name="customAspectRatioNumerator">
          <property name="minimum">
           <number>1</number>
          </property>
          <property name="maximum">
           <number>9999</number>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="customAspectRatioSeparator">
          <property name="text">
           <string>:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSpinBox" name="customAspectRatioDenominator">
          <property name="minimum">
           <number>1</number>
          </property>
          <property name="maximum">
           <number>9999</number>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>Crop:</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QComboBox" name="displayCropMode"/>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="label_6">
        <property name="text">
         <string>Downsampling:</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QComboBox" name="gpuDownsampleMode"/>
      </item>
      <item row="3" column="0" colspan="2">
       <layout class="QGridLayout" name="gridLayout">
        <item row="0" column="0">
         <widget class="QCheckBox" name="displayIntegerScaling">
          <property name="text">
           <string>Integer Upscaling</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QCheckBox" name="displayStretch">
          <property name="text">
           <string>Stretch To Fill</string>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QCheckBox" name="displayLinearFiltering">
          <property name="text">
           <string>Linear Upscaling</string>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QCheckBox" name="internalResolutionScreenshots">
          <property name="text">
           <string>Internal Resolution Screenshots</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox_5">
     <property name="title">
      <string>On-Screen Display</string>
     </property>
     <layout class="QGridLayout" name="formLayout_5">
      <item row="2" column="0">
       <widget class="QCheckBox" name="showOSDMessages">
        <property name="text">
         <string>Show OSD Messages</string>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QCheckBox" name="showSpeed">
        <property name="text">
         <string>Show Emulation Speed</string>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="QCheckBox" name="showFPS">
        <property name="text">
         <string>Show FPS</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QCheckBox" name="showResolution">
        <property name="text">
         <string>Show Resolution</string>
        </property>
       </widget>
      </item>
      <item row="4" column="0">
       <widget class="QCheckBox" name="showCPU">
        <property name="text">
         <string>Show CPU Usage</string>
        </property>
       </widget>
      </item>
      <item row="4" column="1">
       <widget class="QCheckBox" name="showInput">
        <property name="text">
         <string>Show Controller Input</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
